### ISSUE TITLE
[Bug] [Intermittent] check_ip_address failed occasionally

### ISSUE THREAD

**Issue by keirazhang** [vmware/ansible-vsphere-gos-validation#94]
##### SUMMARY

```
2021-07-08 13:07:34,008 | Failed at Play [check_ip_address] **************************

2021-07-08 13:07:34,008 | TASK [Check IP address lists difference] *******************
task path: /root/ansible-vsphere-gos-validation/linux/check_ip_address/validate_ip_address.yml:17
fatal: [localhost]: FAILED! => {
    "assertion": "all_ip_in_guest | difference(all_ip_by_tools) | length == 0",
    "changed": false,
    "evaluated_to": false,
    "msg": "\"VM IP addresses got in guest info: ['10.193.8.193', 'fe80::250:56ff:feb5:6db2'], expected IP addresses got in guest OS: ['10.193.8.193', 'fd01:1:2:2918:250:56ff:feb5:6db2', 'fe80::250:56ff:feb5:6db2']\"\n"
}
```
Rerun could pass. 

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below
ansible-playbook [core 2.11.1] 
  config file = /root/ansible-vsphere-gos-validation/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible-playbook
  python version = 3.9.1 (default, May 19 2021, 10:27:03) [GCC 10.2.0]
  jinja version = 3.0.1
  libyaml = True


```

##### CONFIGURATION

```paste below



```

##### ENVIRONMENT / OS

RHEL 8.4

**Comment by keirazhang** [comment]
This issue was found in https://gosv.svc.eng.vmware.com/view/Regression/job/Ansible_Regression_RHEL_8.4/315/ for the first time, but it cannot be reproduced in following test. So I would prefer to close this issue. If we hit it in future, we will open new issue for tracking and fixing.

**Comment by keirazhang** [comment]
Fix has been merged in #214
